common/test/acceptance/pages/studio/library.py

```diff
@@ -168,6 +169,7 @@ def count(self, count):
         Sets value of children count input
         """
         count_text = self.get_setting_element(self.COUNT_LABEL)
+        count_text.send_keys(Keys.BACK_SPACE)
```

Comment:
@benpatterson Clear method don't clear the input list and script was appending the values with default value e.g if 1 was the default value and we input 10 then It would be 110 instead of 10 that's why I used force clear (Back Space).

It works on firefox very well and I have tested 4-5 times on GC, 3 times on FF. This step works on both.